test/e2e/util/util.go

```diff
@@ -1167,6 +1193,454 @@ func EnsureNetworkPolicies(t *testing.T, ctx context.Context, c crclient.Client,
 	})
 }
 
+// InstallCiliumNetworkPolicies validates that Cilium network policies are properly enforced
+// in ARO HCP guest clusters. This test covers:Verifying Cilium installation
+func InstallCiliumNetworkPolicies(t *testing.T, ctx context.Context, guestClient crclient.Client, hostedCluster *hyperv1.HostedCluster) {
+	t.Run("InstallCiliumNetworkPolicies", func(t *testing.T) {
+		if hostedCluster.Spec.Platform.Type != hyperv1.AzurePlatform {
```

Comment:
I think we could just drop this if statement since the one on L1204 should be sufficient.